Backend/lib/beckn-spec/src/BecknV2/FRFS/Enums.hs

```diff
@@ -91,3 +91,22 @@ instance FromJSON PaymentStatus where
 
 instance ToJSON PaymentStatus where
   toJSON = genericToJSON constructorsWithHyphens
+
+data CancellationType = SOFT_CANCEL | CONFIRM_CANCEL
+  deriving (Eq, Ord, Show, Read, Generic)
+
+data CancellationParams = REFUND | CANCELLATION_CHARGES | BASE_FARE
+  deriving (Eq, Ord, Show, Read, Generic)
+
+data OnCancelOrderStatus = ON_CANCEL_SOFT_CANCEL | ON_CANCEL_CANCELLED
```

Comment:
The first set of enums are being used to exract cancellationParams from OnCancel, the second set it used to detemine the set of actions to take in case of SoftCancel and HardCancel, We are parsing them from order status in OnCancel resp.